gpflow/likelihoods/scalar_continuous.py

```diff
@@ -36,61 +45,92 @@ class Gaussian(ScalarLikelihood):
     likelihood variance by default.
     """
 
-    DEFAULT_VARIANCE_LOWER_BOUND = 1e-6
-
     def __init__(
         self,
-        variance: float = 1.0,
-        variance_lower_bound: float = DEFAULT_VARIANCE_LOWER_BOUND,
+        variance: Optional[ConstantOrFunction] = None,
+        *,
+        scale: Optional[ConstantOrFunction] = None,
+        variance_lower_bound: float = DEFAULT_LOWER_BOUND,
```

Comment:
Would it make sense to leave the name as before, i.e. DEFAULT_VARIANCE_LOWER_BOUND?